lib/bitgo/v1/api.rb

```diff
@@ -436,10 +462,10 @@ def transaction(tx_hash)
 
       # Perform HTTP call
       # path parameter must being with a /
-      def call(method, path, params = {}, parse_response_as_json = true, with_auth_token = true)
+      def call(method, path, params = {}, parse_response_as_json = true, with_auth_token = true, query: {})
```

Comment:
use params consistently everywhere, but make params as queryparms if the request is GET